assert/assertions.go

```diff
@@ -45,6 +45,10 @@ type BoolAssertionFunc func(TestingT, bool, ...interface{}) bool
 // for table driven tests.
 type ErrorAssertionFunc func(TestingT, error, ...interface{}) bool
 
+// PanicAssertionFunc is a common function prototype when validating a panic value.  Can be useful
+// for table driven tests.
+type PanicAssertionFunc func(t TestingT, f PanicTestFunc, msgAndArgs ...interface{}) bool
```

Comment:
A type alias would be enough.
```suggestion
type PanicAssertionFunc = func(t TestingT, f PanicTestFunc, msgAndArgs ...interface{}) bool
```

Reply:
Hi @dolmen thanks for the feedback

Addressed in https://github.com/stretchr/testify/pull/1337/commits/dc1948be0602f49f6de04278bf33a78b1c941486

Reply:
Please also rebase on master as you just added pollution to your patch because of gofmt 1.20 changes.

Reply:
Rebased

Reply:
Thanks!